test/unit/store/student-store.js

```diff
@@ -198,4 +213,30 @@ describe('StudentStore', () => {
             assert.lengthOf(student.items, 1);
         });
     });
+
+    it(`should edit item's duedate in student's list of items`, () => {
```

Comment:
This test shouldn't even be in this PR it some how got merged into this branch from existing-to-longterm. It did find a timezone bug in NEW_CHECKOUT though so that's pretty neat.